ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/NewTabPageViewController.swift

```diff
@@ -421,6 +422,7 @@ class NewTabPageViewController: UIViewController {
     super.viewDidAppear(animated)
 
     reportSponsoredBackgroundEvent(.servedImpression) { [weak self] _ in
+      self?.delegate?.showNewTabTakeoverInfoBarIfNeeded()
```

Comment:
nit: Infobar, please see all InfoBar :-)

Reply:
Discussed in DM and decided to keep it as it is, because chromium uses `InfoBar`